### ISSUE TITLE
Let special-lispy-other-mode decribe bindings

### ISSUE THREAD

**Issue by sooheon** [abo-abo/lispy#238]
`o` or `og` gives messages in minibuffer like `[other]: h j k l SPC g` or `[goto]: d l r p f b q j a e`. It would be useful if it displayed the commands in addition to the keys. @justbur, do you have any advice on how to make these compatible with which-key? The keys are defined [here](https://github.com/abo-abo/lispy/blob/master/lispy.el#L7414), and the defverb macro is [here](https://github.com/abo-abo/lispy/blob/master/lispy.el#L477)

**Comment by abo-abo** [comment]
Defverb is a pre-hydra mechanism. The only reason they're still around is that I haven't had time to update them.

That, and also they're less intrusive than hydras, because they don't resize any windows since they only need one line.  I'll add that feature to hydra at some point. Well, `(setq hydra-lv nil)` already works, but I want something more flexible.

**Comment by justbur** [comment]
If this were a hydra, which-key would be unnecessary so I don't really see a reason to hack something to make it work with which-key in this case. 

In playing with this, one downside of defvarb seems to be that because it uses the echo area the message kept getting clobbered by eldoc for me.

**Comment by sooheon** [comment]
Yeah no need for both hydra and which key. Though it seems to me a lot of these actions are more one-off than repeatable, so which key would be more appropriate.
